file_process/base.py

```diff
@@ -31,7 +31,14 @@ def get_preview_data(self, adata):
         obs_preview = adata.obs.head(n=10)
         return target_names, var_preview, obs_preview
 
-    def model_file_validation(self, adata: AnnData, model_metadata_file: BytesIO):
+    def model_file_validation(self, adata: AnnData, model_metadata_file: Optional[BytesIO] = None):
```

Comment:
It's a little bit confusing: the function is model file validation, but the model meta file is optional. I think it's better we separate the target validation from the validation against the model metadata.